src/hooks/useAccessControl/index.test.jsx

```diff
@@ -51,6 +53,8 @@ describe("useAccessControl", () => {
       .spyOn(accessControlFns, "getAccessControl")
       .mockResolvedValue(accessControl);
     mockedPoliciesContainerUrlHook.mockReturnValue(null);
+    const mockedLegacyAcpHook = jest.requireMock("../useIsLegacyAcp");
```

Comment:
You can just import the hook as normal at the top of the file and then do `const mockedLegacyAcpHook = useIsLegacyAcp`. Then when you mock the return value you don't need to add `default` to it.

Reply:
30bbd48